### PR TITLE
Cart view

### PR DESCRIPTION
**Pull request by miaolphoebe**

### CODE REVIEW

File: client/components/Cart.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  render() {
+    const order = this.props.order;
+    return (
+      <div id="single-product" className="column">
+        <div id="single-product-detail" className="row">
+          <div className="column rm1" key={order.id}>
+            <h1> Shopping Cart</h1>
+            {/* <img width="250px" height="250px" src={order.imageUrl} /> */}
```
Comment (MichelleMullane) [inline client/components/Cart.js:20]: Do you want this to be commented out?

File: client/store/singleOrder.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      const { data } = await axios.get(`/api/orders/${id}`);
+      dispatch(setSingleOrder(data));
+    } catch (err) {
+      console.log(err);
+    }
+  };
+};
+
+const initialState = {
+  id: "",
```
Comment (MichelleMullane) [inline client/store/singleOrder.js:24]: are these keys correct for the order?  They should match the model?

File: script/seed.js [hunk 3]
```diff
@@ -192,11 +188,11 @@ async function seed() {
...
     addresses: {
-      address1: addresses[0], 
-      address2: addresses[1], 
-      address3: addresses[2], 
-      addresses4: addresses[3], 
-      addresses5: addresses[4]
+      address1: addresses[0],
+      address2: addresses[1],
+      address3: addresses[2],
+      addresses4: addresses[3],
```
Comment (MichelleMullane) [inline script/seed.js]: Why change key from "address" to "addresses"?

File: client/components/Cart.js (hunk 1 above)
Comment (miaolphoebe) [reply client/components/Cart.js:20]: Yes. This is for future use.

File: script/seed.js (hunk 3 above)
Comment (miaolphoebe) [reply script/seed.js]: I didn't change anything here, it's just the code editor cleaned up the spacing.

**Review by MichelleMullane — APPROVED** [review]

**Pull request merged**